### PR TITLE
feat: juno analytics

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Enable the Juno's analytics (feature currently in a trial phase) for gix.design.

# Changes

- add dependency `@junobuild/analytics`
- copy worker on post install
- init analytics for the dapp (showcase of the gix-cmp)

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
I'd love to try that. They are on-chain, right?

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -2,6 +2,15 @@
   import Layout from "$lib/components/Layout.svelte";
   import DocsMenu from "$docs/components/DocsMenu.svelte";
   import Toasts from "$lib/components/Toasts.svelte";
+  import { onMount } from "svelte";
+  import { initOrbiter } from "@junobuild/analytics";
+
+  onMount(async () =>
+    await initOrbiter({
+      satelliteId: "nbyi7-6aaaa-aaaal-acjtq-cai",
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/routes/+layout.svelte]: Good idea 👍. Done. I have also disabled the analytics for local development at the same time. Thanks.

**Comment by peterpeterparker** [comment]
> I'd love to try that. They are on-chain, right?

@lmuntaner yes on chain. I'm starting the trial phase to collect data and find what needs to be improved before making it public to everyone. Therefore thought about adding those to gix-cmp as well. It can also be interesting for the team to know if gix.design is used. So win win 😉.

Capture d’écran 2023-09-04 à 07 22 00

**Comment by lmuntaner** [comment]
It looks super cool!!

We could even consier adding it to nns-dapp, right?

**Comment by peterpeterparker** [comment]
It's definitely achievable.

It will be integrated within Juno, but from a technical standpoint, I've designed it to be agnostic. That was it can potentially become a standalone product and an option for any dapps.

**Pull request merged**